### PR TITLE
Add some options for webservice uploading

### PR DESCRIPTION
**Pull request by rkdarst**
Hi,

I'm not sure if you will want this, but these patches add two variables `webservice_stateless` and `webservice_only`, which if combined will give this the behavior of the iOS app and remove all data once it is uploaded.  This is to user-visible anywhere, so is only activated if you like with a server that supports it.  The people running the study have to be clever enough to know that it does not work with all probes or plugins, and know the implications of not doing a three-phase sync.  I would consider it advanced use.

Note that indentation is wrong in this patch (to minimize the diff).  If you want to accept it, I will fix up the indentation first and then you can take it.

Thanks,

- Richard

Commit message:

- webservice_stateless: don't /create_table: in this case, the server
  saves all raw data and figures out what to do.
- webservice_only: removes data once it is uploaded.  May be
  incompatible with some sensors, but it seems to work so far.
- webservice_only and webservice_stateless: if both are given, then do
  not do /latest, and instead upload all data in the database.
- This patch introduces improper indentation, but it minimizes the
  divergence from upstream and is localized.

### CODE REVIEW

File: aware-core/src/main/java/com/aware/Aware_Preferences.java [hunk 1]
```diff
@@ -465,6 +465,16 @@
     public static final String FREQUENCY_WEBSERVICE = "frequency_webservice";
 
     /**
+     * AWARE webservice only: remove local data always
+     */
+    public static final String WEBSERVICE_ONLY = "webservice_only";
```
Comment (denzilferreira) [inline aware-core/src/main/java/com/aware/Aware_Preferences.java]: Maybe rename this to: WEBSERVICE_NO_LOG_HISTORY ? Webservice only reads to me as no local storage and constantly sending data remotely in real-time, which is not the case.

**Comment by rkdarst** [comment]
This is a good point!  I have also been thinking that "stateless" is not a very useful name.  How about "webservice_simple" and "webservice_remove_history" ?  I'll make new patch tonight.

**Comment by rkdarst** [comment]
Updated.  Renamed variables, fixed a bug, tested this actual branch, and fixed indention.

**Comment by rkdarst** [comment]
Just to be clear: this means that you want the options visible in the UI ?

**Comment by denzilferreira** [comment]
Yes :) inside the webservice options.

Denzil;

**Comment by rkdarst** [comment]
What about adding an "Advanced options" at the very bottom?  We will probably soon start having options which most people shouldn't need to adjust for normal use.  I'll put these two options there, and "debug", "logcat tag", and these there to start.

**Comment by denzilferreira** [comment]
Maybe developer options is better. Thanks!

Denzil;

**Pull request merged**